src/euphorie/client/browser/locking.py

```diff
@@ -34,6 +34,20 @@ def is_locked(self):
         """Return whether the session is locked."""
         return self.context.session.is_locked
 
+    def is_validated(self):
+        """Return whether the session is locked."""
```

Comment:
```suggestion
        """Return whether the session is validated."""
```
... or is locked equivalent to validated?